cves/kernel/CVE-2022-1671.yml

```diff
@@ -450,7 +464,12 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    There was a lack of separation of concerns within the network and server
+    portion of the project. This caused certain files to be overblown and
+    unreadable as there is just so much that is handled. This was a lapse in
+    judgement for the developers as they should have figured it was best to
+    handle certain data in their own modules.
```

Comment:
Naming the principle here would be good. Maybe "The developers violated principles of high cohesion and low coupling"? or whatever else.